### DISCUSSION TITLE
How do I add more than one "SmallEyes" button? 如何添加多個“小眼睛”按鈕？

### DISCUSSION THREAD

**Discussion by nutlost** [barry-ran/QtScrcpy#618] [General]
I can create many common buttons except "SmallEyes". Anyone know how to make more than 1 button for "small eyes"? I don't know.

我可以創建一個通用按鈕。但除了“SmallEyes”的按鈕 如何為這個“小眼睛”創建多個按鈕？

![image](https://user-images.githubusercontent.com/106454766/171431395-41b192cf-29d6-44ab-8796-cd5066e90366.png)

**Comment by LalatinaBD** [comment]
@nutlost did you get any solution? I tried many things but cant add more than 1 smalleyes.

**Comment by tematou** [comment]
what game do you want to play?
